### PR TITLE
Initial codeowners and maintainers

### LINKED ISSUE #3: Update CODEOWNERS and MAINTAINERS.md initially
**Problem description**

CODEOWNERS and MAINTAINERS.md are not yet set. 

**Expected action**

Needed data collected within https://github.com/camaraproject/APIBacklog/issues/60
Both files updated

**Additional context**

### PR DESCRIPTION
**Pull request by tlohmar**
#### What type of PR is this?

Add one of the following kinds:
* subproject management

#### What this PR does / why we need it:
sets the intial codeowners and maintainers

#### Which issue(s) this PR fixes:

<!-- Automatically closes linked issue when PR is merged.

Fixes #3

#### Changelog input

```
 release-note

```

#### Additional documentation 

This section can be blank.

```
docs

```

### CODE REVIEW

File: MAINTAINERS.MD [hunk 1]
```diff
@@ -1,3 +1,7 @@
-| Org                    | Name                                      | GitHub Username        |
-| -----------------------| ------------------------------------------| -----------------------|
-| §company§ | §name§ | §github_username§ |
+| Org                    | Name                                      | Role | GitHub Username        |
+| -----------------------| ------------------------------------------| ---| -----------------------|
+| Ericsson | Thorsten Lohmar | Condeowner & Maintainer | @tlohmar |
+| Nokia | Peter Kovacs | Condeowner & Maintainer | @PeterKovacs2 |
+| Telefonica| Jorge Garcia Hospital | Condeowner & Maintainer | @jgarciahospital |
+| Vodafone | Eric Murray | Condeowner | @eric-murray |
+| Vodafone | Steve Vickers | Maintainer | @SteveV-Vodafone |
```
Comment (hdamker) [inline MAINTAINERS.MD:7]: Please stay with the given format of the files. Maintainers go here, Codeowners have to go into the CODEOWNERS file. The roles are then implicit given.

```suggestion
| Org                    | Name                                      | GitHub Username        |
| -----------------------| -----------------------------------------| -----------------------|
| Ericsson | Thorsten Lohmar | @tlohmar |
| Nokia | Peter Kovacs | @PeterKovacs2 |
| Telefonica| Jorge Garcia Hospital |  @jgarciahospital |
| Vodafone | Eric Murray | @eric-murray |
| Vodafone | Steve Vickers |  @SteveV-Vodafone |
```

**Review by hdamker — CHANGES_REQUESTED** [review]
The information has to be distributed to the two files CODEOWNERS (used by GitHub to enforce branch protection) and MAINTAINERS.md (format is consistent across all repositories). I added the update of the CODEOWNERS to the PR and suggested the correct format of MAINTAINERS.md above.

I also invited @PeterKovacs2 and @SteveV-Vodafone to the CAMARA organization. As soon as they have accepted the invitation we can go forward here.

**Review by tlohmar — COMMENTED** [review]
thx for the correction.

**Review by ThorstenL2024 — APPROVED** [review]
OK

File: MAINTAINERS.MD (hunk 1 above)
Comment (hdamker) [inline MAINTAINERS.MD:7]:
```suggestion
| Org                   | Name                              |GitHub Username        |
|---------------|------------------------| ----------------------|
| Ericsson | Thorsten Lohmar  | @tlohmar |
| Nokia | Peter Kovacs | @PeterKovacs2 |
| Telefonica| Jorge Garcia Hospital | @jgarciahospital |
| Vodafone | Steve Vickers | @SteveV-Vodafone |
```

**Pull request merged**